utils/Assets.ts

```diff
@@ -65,10 +66,19 @@ export class Assets {
         txs.push(Assets.issueToken(user, currencyValues[currency]));
       }
       const result = await Sudo.batchAsSudoFinalized(...txs);
-      const assetIds: BN[] = result
+      assetIds = result
         .filter((X) => X.method === "Created")
         .map((t) => new BN(t.eventData[0].data.toString()));
       const addInfos: Extrinsic[] = [];
+      while (assetIds[0].toNumber() < 8) {
```

Comment:
why this? 
what if 2 concurrent tests run this in parallel?  I think it does not make sense to have it here.

Reply:
Because now this function creates tokenId = 1, 2, etc. I made this as a defence against this problem, but I can change it